TrainingOOP-PHP001/studentManagementOOP.php

```diff
@@ -0,0 +1,125 @@
+<?php
+namespace TrainingOOP1;
+
+class Student {
+    public $name;  
+    public $score; 
+
+    /**
+     * Student class constructor
+     *
+     * @param string $name The name of the student
+     * @param float $score The score of the student
+     */
+    public function __construct($name, $score) { 
+        $this->name = $name;   
+        $this->score = $score; 
+    }
+}
+
+/**
+ * Class for managing students.
+ */
+class StudentManagement {
+    private $listStudent = []; // Storage array for students
```

Comment:
Tên biến array nên có số nhiều
```suggestion
    private $listStudents = []; // Storage array for students
```